docs/guides/implementation-guides/how-to-implement-dependency-injection.md

```diff
@@ -100,6 +100,8 @@ public partial class App : Application
         // Without this line you will get duplicate validations from both Avalonia and CT
         BindingPlugins.DataValidators.RemoveAt(0);
 
+        // `ServiceManager` retrieve an instance of `MainViewModel`
```

Comment:
I assume this is a copy paste error, but is there a reason you have markdown code blocks inside C# comments?